neqo-transport/src/tparams.rs

```diff
@@ -722,10 +733,29 @@ impl TransportParametersHandler {
     pub const fn remote_handshake(&self) -> Option<&TransportParameters> {
         self.remote_handshake.as_ref()
     }
+
+    /// Filter to retain only those transport parameters that are necessary for an outer `ClientHello`.
+    const fn filter_ch_outer(tp: TransportParameterId, _v: Option<&TransportParameter>) -> bool {
```

Comment:
It might be a good idea to add the analysis from the first comment in the PR here?

Reply:
I was in the process of doing that as you asked :)  kazuho suggested that we also share with the QUIC mailing list, which I'll do as well.